invenio_app_ils/config.py

```diff
@@ -187,8 +188,15 @@ def _(x):
         "schedule": timedelta(hours=3),
         "args": [("record-view-agg", "file-download-agg")],
     },
+    "clean_locations_past_closures_exceptions": {
+        "task": "invenio_app_ils.closures.tasks.clean_locations_past_closures_exceptions",
+        "schedule": timedelta(weeks=4),
```

Comment:
I am still doubting about this, ping other reviewers @kprzerwa @topless @KonstantinaStoikou.
Isn't it maybe weird that in June I cannot see the closures I had set up for last April or May? They will be deleted.
Wondering if we should run this once a year? Which it also sounds weird! :D